pack/nodejs/helm.ts

```diff
@@ -0,0 +1,715 @@
+import { execSync } from "child_process";
+import * as fs from "fs";
+
+import * as k8s from "./index";
+import * as pulumi from "@pulumi/pulumi";
+import * as shell from "shell-quote";
+import * as tmp from "tmp";
+import * as yaml from "js-yaml";
+
+export namespace v2 {
+    export interface ChartOpts {
+        repo: string;
+        chart: string;
+        version: string;
+
+        values?: any;
+        transformations?: transform.Transform[];
+        fetchOpts?: FetchOpts;
+    }
+
+    // A CustomResource representing a Helm Chart.
```

Comment:
This is actually a `ComponentResource` - though for the docs might be better to just say a "Component".